package.json

```diff
@@ -58,17 +55,16 @@
     "jest": "^26.4.2",
     "jest-styled-components": "^7.0.3",
     "lint-staged": "^10.4.0",
-    "prettier": "^2.1.2",
-    "react-docgen-typescript-loader": "^3.7.2",
-    "storybook-addon-jsx": "^7.3.4"
+    "prettier": "^2.0.5",
+    "react-docgen-typescript-loader": "^3.7.2"
   },
   "dependencies": {
     "date-fns": "^2.16.1",
     "react": "^16.13.1",
     "react-dom": "^16.13.1",
+    "snyk": "^1.404.0",
```

Comment:
Not fully relate but wouldn't snyk rather be a devDependencies? 🤔 not fully sure what it actually does as a JS dependency here...
Can also be ignore not to add risk to the PR as these are always tricky...